packages/mui-material/src/SwipeableDrawer/SwipeableDrawer.d.ts

```diff
@@ -2,6 +2,20 @@ import * as React from 'react';
 import { DrawerProps } from '../Drawer';
 
 export interface SwipeableDrawerProps extends Omit<DrawerProps, 'onClose' | 'open'> {
+  /**
+   * Callback to determine what children of the drawer the user can use to drag open the drawer.
+   * Can be a custom function to control the behavior.
+   * Set or return true / false to allow / disallow the swipe event.
+   *
+   * @param {TouchEvent} event The 'touchstart' event
+   * @param {HTMLDivElement} swipeArea The swipe area element
+   * @param {HTMLDivElement} paper The drawer's paper element
+   *
+   * @default false
+   */
+  AllowSwipeInChildren?:
```

Comment:
Use camel case in prop names

```suggestion
  allowSwipeInChildren?:
```